### ISSUE TITLE
Regression: Link hints must be typed uppercase to match

### ISSUE THREAD

**Issue by bgmort** [philc/vimium#2749]
After the latest update to Chrome on a Mac (1.61.1), link hints display as uppercase, and don't match unless they are typed as uppercase.  This really slows down the flow for following links.

**Comment by mrmr1993** [comment]
Have you tried resetting "Characters used for link hints" setting to the default value?

You can do this by opening the options page (from chrome://extensions), pressing the "Show Advanced Options" button, clearing the textbox for that option, and then pressing "Save Changes".

**Comment by bgmort** [comment]
No luck.  I forgot that numbers are used for link hints (also too slow), but after resetting to the default, then putting my hint characters back in (zxcvbghjkl), they still display and match as uppercase letters only.

**Comment by smblott-github** [comment]
Have you selected *Use the link's name and characters for link-hint filtering* on the options page?  If so, then you have to hold shift to select hint characters (because lower-case characters match a link's text).

If not, then hints are displayed upper-case but you do not need to hold down `Shift`.

**Comment by bgmort** [comment]
Yes, that option was checked.  Until the most recent update, though, both link text and link hints could be matched with lowercase letters.  In my opinion that's a nice feature that should be kept around.  It's also nice when the case of the hint text matches what has to be typed.

**Comment by smblott-github** [comment]
> Until the most recent update, though, ...

I believe that was an accidental feature.

> It's also nice when the case of the hint text matches what has to be typed.

Internally, the hint text is always handled in lower case, and always has been.
